### PR TITLE
Upgrade dependencies and some cleanup

### PR DESCRIPTION
**Pull request by Dignifiedquire**
- [x] Circle config (lint & build)
- [x] Travis config (lint & build)
- [x] Readme badges
- [x] Update dependencies
- [x] Remove `static/fonts` in favor of `node_modules/font-awesome` (Resolves #72)
- [x] Fix geo-ipfs lookup

### CODE REVIEW

File: js/getlocation.js [hunk 1]
```diff
@@ -13,8 +13,10 @@ function isLocal (address) {
 
 var getLocation = module.exports = function (ipfs, multiaddrs, cb) {
   if (multiaddrs.length === 0) return cb(null, null)
-  var address = multiaddrs[0].split('/')[2]
-  if (isLocal(address)) return getLocation(ipfs, multiaddrs.slice(1), cb)
+  var current = multiaddrs[0].split('/')
+  var address = current[2]
+  // No support for ip6 lookups currently
+  if (isLocal(address) || current[1] === 'ip6') return getLocation(ipfs, multiaddrs.slice(1), cb)
```
Comment (diasdavid) [inline js/getlocation.js]: Can we always do curly ifs? Actually, my standard with vim's syntastic always warns me of not doing curly ifs, doesn't it  happen for you?

**Comment by diasdavid** [comment]
LGTM :) Thank you, @Dignifiedquire ! :)

File: js/getlocation.js (hunk 1 above)
Comment (Dignifiedquire) [inline js/getlocation.js]: Nope as far as I know standard only errors on multiline ifs without curlys, which is what I adopted, no curlys if one line, curlys for multiline. Sounds like your vim plugin needs some updates ;)

But I don't have strong feelings about it, I just don't like curlys if I can avoid them ;)

Comment (diasdavid) [inline js/getlocation.js]: sounds good, going to merge then :)

**Pull request merged**